pages/sidecar.rst

```diff
@@ -148,170 +159,98 @@ system service. We just need the binaries installed on the system::
-
-For the Beats platform you can enable each Beat individually, e.g on a Windows host with Filebeat and Winlogbeat enabled use a configuration like this::
-
-    server_url: http://10.0.2.2:9000/api/
-    update_interval: 30
-    tls_skip_verify: true
-    send_status: true
-    list_log_files:
-    node_id: graylog-collector-sidecar
-    collector_id: file:C:\Program Files\graylog\collector-sidecar\collector-id
-    cache_path: C:\Program Files\graylog\collector-sidecar\cache
-    log_path: C:\Program Files\graylog\collector-sidecar\logs
-    log_rotation_time: 86400
-    log_max_age: 604800
-    tags:
-      - windows
-      - apache
-      - redis
-    backends:
-        - name: winlogbeat
-          enabled: true
-          binary_path: C:\Program Files\graylog\collector-sidecar\winlogbeat.exe
-          configuration_path: C:\Program Files\graylog\collector-sidecar\generated\winlogbeat.yml
-        - name: filebeat
-          enabled: true
-          binary_path: C:\Program Files\graylog\collector-sidecar\filebeat.exe
-          configuration_path: C:\Program Files\graylog\collector-sidecar\generated\filebeat.yml
-
-On the server side the collector plugin is caching the requested configuration in memory. By default up-to 100 entries are stored for 1 hour.
-If you wish to change that, add to your server configuration::
-
-    collector_sidecar_cache_time = 2h
-    collector_sidecar_cache_max_size = 500
+    C:\Program Files\Graylog\sidecar\sidecar.yml
+
+Most configuration parameters come with built-in defaults. The only mandatory parameters are ``server_url`` and ``server_api_token``.
+
+.. |br| raw:: html
+
+     <br>
+
+
++-------------------------------------+---------------------------------------------------------------------------------------------------------------------+
+| Parameter                           | Description                                                                                                         |
++=====================================+=====================================================================================================================+
+| server_url                          | URL to the Graylog API, e.g. ``http://192.168.1.1:9000/api/``                                                       |
++-------------------------------------+---------------------------------------------------------------------------------------------------------------------+
+| server_api_token                    | The API token to use to authenticate against the Graylog server API. |br|                                           |
+|                                     | e.g ``1jq26cssvc6rj4qac4bt9oeeh0p4vt5u5kal9jocl1g9mdi4og3n``  |br|                                                  |
+|                                     | The token is not optional and needs to be configured.                                                               |
```

Comment:
> not optional 

Maybe `mandatory` is the better choice? 

People will not notice the `not` and read optional...